src/main/java/me/zeroeightsix/fiber/builder/constraint/ConstraintsBuilder.java

```diff
@@ -30,9 +32,23 @@ public ConstraintsBuilder(List<Constraint<? super T>> sourceConstraints, Class<T
 		return this;
 	}
 
+	public ConstraintsBuilder<T> minStringLength(int min) {
+		addStringLengthLowerBound(min);
+		return this;
+	}
+
+	public ConstraintsBuilder<T> maxStringLength(int min) {
+		addStringLengthUpperBound(min);
+		return this;
+	}
```

Comment:
Can these methods also be implemented for the composite constraint builder?

(+ the same naming suggestion applies to these)